### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by keroloussamy**
In this release, we made the following 

1. #20 
2. #19 
3. #18 
4. #17
5. #16 
6. #15 
7. #14 
8. #12 
9. #11 
10. #9 
11. #8 
12. #6 
13. #5 
14. #3 
15. #2

### CODE REVIEW

File: src/API.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const likeURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/lCNzVmZgTty5Cce1TAf3/likes/';
+
+async function fetchMeals(categoryName) {
```
Comment (rloterh) [inline src/API.js]: - [x] It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: src/commentsPopup.test.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { addCommentCount } from './commentsPopup.js';
+
```
Comment (rloterh) [inline src/commentsPopup.test.js:2]: - [x] Kindly ensure that the test case covers all the pure functions used in this project

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
-# JavaScript-Capston-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (rloterh) [inline README.md:1]: How to run test cases is not included in your ReadMe, and it needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [x] _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [x] The _code_ aspects in each procedure should be quoted with **backticks**.
- [x] Include necessary screenshots, project demo links where applicable so as to give a birds-eye view of your project
- [x] All _typos_ **should be corrected**. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Omar & Kerolous, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- Nice commit messages
- Good presentation of the project
- Promise feature properly handled
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/API.js (hunk 1 above)
Comment (keroloussamy) [reply src/API.js]: Done

File: README.md (hunk 3 above)
Comment (keroloussamy) [reply README.md:1]: Done

File: src/commentsPopup.test.js (hunk 2 above)
Comment (keroloussamy) [reply src/commentsPopup.test.js:2]: Done

**Review by HillaryOkello — APPROVED** [review]
# Hi Team, 

Nice work implementing the changes suggested by the previous reviewer 👏🏻 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉. Keep up the good work 💪🏻 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**